test/performance-test-suite/pkg/runner/benchmarks.go

```diff
@@ -24,21 +24,63 @@ import (
 func getBenchmarksToRun() []benchmarks.Benchmark {
 	return []benchmarks.Benchmark{
 		writetxs.NewBenchmark(writetxs.Config{
-			Name:       "Write TX/s",
+			Name:       "Write TX/s - no replicas",
 			Workers:    30,
 			BatchSize:  1,
 			KeySize:    32,
 			ValueSize:  128,
 			AsyncWrite: true,
+			Replica:    "",
 		}),
 
 		writetxs.NewBenchmark(writetxs.Config{
-			Name:       "Write KV/s",
+			Name:       "Write KV/s - no replicas",
 			Workers:    30,
 			BatchSize:  1000,
 			KeySize:    32,
 			ValueSize:  128,
 			AsyncWrite: true,
+			Replica:    "",
+		}),
+
+		writetxs.NewBenchmark(writetxs.Config{
+			Name:       "Write TX/s - one async replica",
+			Workers:    30,
+			BatchSize:  1,
+			KeySize:    32,
+			ValueSize:  128,
+			AsyncWrite: true,
+			Replica:    "async",
+		}),
+
+		writetxs.NewBenchmark(writetxs.Config{
+			Name:       "Write KV/s - one async replica",
+			Workers:    30,
+			BatchSize:  1000,
+			KeySize:    32,
+			ValueSize:  128,
+			AsyncWrite: true,
+			Replica:    "async",
+		}),
+
+		writetxs.NewBenchmark(writetxs.Config{
+			Name:       "Write TX/s - one sync replica",
+			Workers:    30,
+			BatchSize:  1,
+			KeySize:    32,
+			ValueSize:  128,
+			AsyncWrite: true,
```

Comment:
@jeroiraz wdyt, having those tests with asyncwrite=true would be more reasonable ?

Reply:
performance changes a lot when using sync vs async writes, ideally both cases could be covered. If not, I'd suggest to use sync writes i.e. `AsyncWrite: false,` so to be sure we are not downgrading performance with newer changes

Reply:
I've created https://github.com/vchain-us/codenotary-cloud/issues/2401 to add these scenarios when we can.